test/AutoRest.TestServer.Tests/SpreadClientTest.cs

```diff
@@ -35,5 +35,146 @@ public void SpreadModelMethod()
                 Assert.AreEqual("Thing", parameters[0].ParameterType.Name);
             }
         }
+
```

Comment:
My understanding is this impacts the payload. So we need E2E tests. This is an example: https://github.com/Azure/autorest.csharp/blob/d63b897ea987385356d71e225fa38db9e146e8bd/test/CadlRanchProjects.Tests/LroBasic-Cadl.cs

Reply:
hello @pshao25 Yes, the request body payload is important, but now there is no way to get the internal request body payload for test. Do you know any available way?

Reply:
1. Writing mock api which reflects your cadl under test\CadlRanchMockApis\src.
2. Writing unit tests under test\CadlRanchProjects.Tests, LroBasic-Cadl.cs is an example.

Reply:
Thanks @pshao25 I added the CadlRanch test.